index.md

```diff
@@ -5,7 +5,7 @@
 
 Home <welcome>
 Examples <https://www.pymc.io/projects/examples/en/latest/gallery.html>
-User guide <https://www.pymc.io/projects/docs/en/latest/learning.html>
+Learn <https://www.pymc.io/projects/docs/en/latest/learn.html>
```

Comment:
```suggestion
Learn <learn>
```

This needs to be a cross reference, otherwise it breaks the versioning. i.e. being on 4.0 docs after 4.1 is released and clicking on learn will change the version without notice